pages/common/gpg-zip.md

```diff
@@ -0,0 +1,15 @@
+# gpg-zip
+
+> Encrypt files and directories in an archive using GPG.
+
+- Encrypt a directory into archive.gpg using a passphrase:
+
+`gpg-zip --symmetric --output {{archive.gpg}} {{dirname}}`
+
+- Decrypt archive.gpg into a directory of the same name:
+
+`gpg-zip --decrypt {{archive.gpg}}`
```

Comment:
Same here. `archive.gpg` -> `path/to/archive.gpg` please :slightly_smiling_face: